SeriesHW.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<ul><li>writing it in python 1st is actually a good idea. And also very close to reality, since usually you figure out it is too slow in python and you have to use something else. However then you already got some working examples you can compare it to!</li><li>what didn't you do a function for the uint Datatype?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>